distsql/request_builder.go

```diff
@@ -55,7 +57,7 @@ func (builder *RequestBuilder) SetIndexRanges(sc *stmtctx.StatementContext, tid,
 	if builder.err != nil {
 		return builder
 	}
-	builder.Request.KeyRanges, builder.err = IndexRangesToKVRanges(sc, tid, idxID, ranges)
+	builder.Request.KeyRanges, builder.err = IndexRangesToKVRanges(sc, tid, idxID, ranges, nil)
```

Comment:
Why set fb to nil here?

Reply:
This function is only used for analyze, so I set nil here.